instrumentation/kafka-streams/src/test/java/brave/kafka/streams/ITKafkaStreamsTracing.java

```diff
@@ -263,6 +264,61 @@ record -> {
     streams.cleanUp();
   }
 
+  @Test void injectsInitContextOnForward() {
+    String inputTopic = testName + "-input";
+
+    BlockingDeque<TraceContext> contexts = new LinkedBlockingDeque<>();
+
+    StreamsBuilder builder = new StreamsBuilder();
+    builder.<String, String>stream(inputTopic)
+      .process(kafkaStreamsTracing.process("forward",
+        () -> new ContextualProcessor<String, String, String, String>() {
+          @Override public void process(Record<String, String> record) {
+            context().forward(record);
+          }
+        }))
+      .process(kafkaStreamsTracing.process("check",
+        () -> (Processor<String, String, String, String>) record -> {
+          contexts.add(currentTraceContext.get());
```

Comment:
added this here as otherwise you don't know for sure that both it had the right parent and also the wrapped processor had the correct local span.